From: Adrian Bunk <bunk@debian.org>
Date: Tue, 16 Oct 2018 10:49:01 +0200
Subject: Fix a parallel build FTBFS

Files were sometimes written to trans/ before that directory was created.
Based on similar patch for tuxpaint-config
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 730ed91..afaea32 100644
--- a/Makefile
+++ b/Makefile
@@ -410,7 +410,7 @@ endif
 
 # Build the translation files for gettext
 
-$(MOFILES): trans/%.mo: src/po/%.po  
+$(MOFILES): trans/%.mo: src/po/%.po trans
 	msgfmt -o $@ $<
 
 .PHONY: translations
